doc/ReconstructingMeasurements.rst

```diff
@@ -83,59 +83,42 @@ id for each object.
+``addParents`` flag (False by default) will add these parents to the idList.
 
 .. code-block:: python
 
-    # Continuing from above again
-    from timeit import Timer
-    from collections import namedtuple
-
-    timingResult = namedtuple("TimingResult",
-        ['niter', 'total', 'min', 'median', 'mean'])
-
-    # Define a convenient function for timing
-    # This excludes the time spent rebuilding the NoiseReplacer
-    def timeRunPlugins(repeat=2):
-        timer = timeit.Timer(
-            "measTask.runPlugins(noiseReplacer, newSrcCatalog, exposure)",
-            setup="from lsst.meas.base.measurementInvestigationLib "
-                  "import rebuildNoiseReplacer;"
-                  "noiseReplacer = rebuildNoiseReplacer(exposure, srcCat)",
-            globals={'exposure':exposure, 'srcCat':srcCat, 'measTask': measTask,
-                    'newSrcCatalog':newSrcCatalog}
-        )
-        times = timer.repeat(repeat,1)
-        result = timingResult(repeat, np.sum(times), np.min(times),
-                              np.median(times), np.mean(times))
-        return result
-
+    import numpy as np
     # Get only the child ids
     idsToRerun = srcCat["id"][srcCat[srcCat.getParentKey()] != 0]
 
-    # Previously, it would skip all children and finish instantaneously
+    # Previously, it would skip all children - only the noiseReplacer takes time
     newSrcCatalog = makeRerunCatalog(
         schema, srcCat, idsToRerun, fields=fields, resetParents=False
     )
-    print("Old (resetParents=False): ", timeRunPlugins())
+    measTask.runPlugins(rebuildNoiseReplacer(exposure, srcCat), newSrcCatalog, exposure)
+    # None of these objects have centroids
+    print(len(newSrcCatalog), np.sum(np.isnan(newSrcCatalog["base_NaiveCentroid_x"])))
 
-    # The new default resets parents and takes a couple of seconds
-    newSrcCatalog = makeRerunCatalog(schema, srcCat, idsToRerun, fields=fields)
-    print("New (default resetParents=True): ", timeRunPlugins())
+    # resetParents=True (default) resets parents and takes a few seconds longer
+    newSrcCatalog = makeRerunCatalog(
+        schema, srcCat, idsToRerun, fields=fields, resetParents=True
+    )
+    measTask.runPlugins(rebuildNoiseReplacer(exposure, srcCat), newSrcCatalog, exposure)
+    # Now none of the objects have nan centroids
+    print(len(newSrcCatalog), np.sum(np.isnan(newSrcCatalog["base_NaiveCentroid_x"])))
 
-    # Running the parents as well takes a little longer
+    # Running the parents as well with addParents=True takes a little longer
     newSrcCatalog = makeRerunCatalog(
-        schema, srcCat, idsToRerun, fields=fields, addParents=True,
-        resetParents=False
+        schema, srcCat, idsToRerun, fields=fields, addParents=True
     )
-    print("New (addParents=True, resetParents=False): ", timeRunPlugins())
+    measTask.runPlugins(rebuildNoiseReplacer(exposure, srcCat), newSrcCatalog, exposure)
```

Comment:
My only last comment will be to add a code comment explaining the expected output, aka with parents more sources will be in the output, like you did in the above two cases